clio-util/src/main/scala/org/broadinstitute/clio/util/model/ServiceAccount.scala

```diff
@@ -0,0 +1,38 @@
+package org.broadinstitute.clio.util.model
+
+import com.google.auth.oauth2.ServiceAccountCredentials
+
+import scala.collection.JavaConverters._
+
+import java.net.URI
+
+/**
+  * Representation of service-account JSON produced by GCloud,
+  * used for persisting Clio updates to cloud storage / for
+  * communicating through Clio's OpenIDC proxy.
+  */
+case class ServiceAccount(authProviderX509CertUrl: URI,
```

Comment:
Not all declared arguments are used. It it required that we declare them to avoid errors when converting from JSON?

Reply:
Yeah, I was getting errors about leftover fields from Circe. I'll add a note in the scaladoc.